File: week#1/mario-more/mario.c
//print our two pyramids facing each other using loop.

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    while (height < 1 || height > 8)
    {
        height = get_int ("Heigh: ");
    }
    for (int i = 0; i < height; i++)
    {
        for(int d = height -1; d > i; d--)
        {
            printf (" ");
        }
        printf ("#");

        for(int j = 0 ; j < i; j++)
        {
            printf ("#");
        }
        printf ("  ");
        printf ("#");
        for(int j = 0 ; j < i; j++)
        {
            printf ("#");
        }
        printf ("\n");
    }

}

File: week#6/sentimental-mario-less/mario.py
from cs50 import get_int

height = get_int("Height: ")

while height <= 0 or height >= 9:
    height = get_int("Height: ")

for i in range(height):
    j = height
    while (j > i + 1):
        print (" ", end="")
        j -= 1
    for k in range(i + 1):
        print("#", end="")
    print()
